jinn/tasks/__init__.py

```diff
@@ -0,0 +1,65 @@
+from invoke import ctask as task
+from invoke import Collection
+from invoke.tasks import call
+
+from jinn import helpers
+
+from . import build, docs, test
+
+
+@task(name='clean-python')
+def clean_python(ctx):
+    """Remove Python file artifacts."""
+    ctx.run('find . -name \'*.pyc\' -delete')
+    ctx.run('find . -name \'*.pyo\' -delete')
+    ctx.run('find . -name \'*~\' -delete')
+    ctx.run('find . -name \'__pycache__\' -delete')
+
+
+@task(pre=[build.clean, call(docs.clean, builddir='_build'), clean_python, test.clean])
+def clean(ctx):
+    """Remove all build, test, coverage and Python artifacts."""
+    pass
+
+
+@task(name='clean-backups')
+def clean_backups(ctx, force=False):
+    """Remove backup files."""
+    if not force:
+        answer = helpers.confirmation_prompt("Do you want to remove all backup files?")
+    if force or answer:
+        ctx.run('find . -name \'*~\' -delete')
+        ctx.run('find . -name \'*.orig\' -delete')
+        ctx.run('find . -name \'*.swp\' -delete')
+
+
+@task(name='clean-bundles')
+def clean_bundles(ctx):
+    """Remove webpack development bundles."""
+    ctx.run('rm -f {ctx.pkg_name}/webpack-stats-development.json'.format(ctx=ctx))
+    ctx.run('rm -fr {ctx.pkg_name}/static/bundles-development/'.format(ctx=ctx))
+
+
+@task
+def develop(ctx):
```

Comment:
This task does neither use a ``constraints.pip`` file, nor is it configurable to use one.